### PR TITLE
Emily's updates to ReadMe.

### PR DESCRIPTION
**Pull request by emilyannmitchell**
I left some comments in there that I set off with HTML tags <!--EM.

My one big problem is that I don't see where the reader is accessing COVID-19 information. Covid-19 isn't even mentioned until the second-to-last step where the reader names their chatbot. Is that part of the JSON file? We need to be mentioning that more if that's the heading in the tutorial. It's not clear to me reading these steps what this has to do with COVID-19. 

Also, note my question about Postman. That's just sort of randomly introduced. Should it be in the prereqs?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,457 +90,467 @@ type: tutorial
...
 
-![ocp-eddidashboard](doc/source/images/eddidashboard.png)
+    ![ocp-eddidashboard](images/eddidashboard.png)
 
 ### Step 4: Create a chatbot in E.D.D.I
 
-- Now, let's create a chatbot in E.D.D.I from scratch.
+Now, let's create a chatbot in E.D.D.I from scratch.
+
+To create a chatbot, you will complete these steps: <!--EM: I feel like, at the very least, we need to link people to where they can learn more about these concepts (regular dictionary, behavior rules, output sets). Is this common vocab for developing chatbots? Why would readers need to develop these things? This feels so advanced and I'm worried about just dropping our readers into this without more explanatory text -->
```
Comment (RahulReddyRavipally) [inline README.md]: **Is this common vocab for developing chatbots?**
Yes, "dictionary, behavior rules, output sets" are common terminologies. Any person/developer dealing with the data would know it. 

**Why would readers need to develop these things?**
Readers would need to develop these things in-order to setup the chatbot from scratch. Infact these detailed steps will be useful for reader to modify the application according to their need.

File: README.md [hunk 2]
```diff
@@ -90,457 +90,467 @@ type: tutorial
...
+1. Create a chatbot
 
-- In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
+In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
 
-#### Step 4.1: Create Regular Dictionary
+#### Step 4.1: Create a regular dictionary
 
-- We create regular dictionaries in order to store custom words and phrases.
+*Regular dictionaries* are used store custom words and phrases. <!--EM: We should clarify this. Is that in machine learning world? AI world?->
```
Comment (RahulReddyRavipally) [inline README.md]: **Is that in machine learning world? AI world?**
Didn't really understand your question but, AI is an umbrella term for Machine Learning. Machine Learning is a part of AI. We can use AI world, it gives broader perspective .

File: README.md [hunk 3]
```diff
@@ -90,457 +90,467 @@ type: tutorial
...
 
-- Make a `POST` to `<eddi-url>/regulardictionarystore/regulardictionaries` with a JSON in the body as follows.
+- Make a `POST` to `<eddi-url>/regulardictionarystore/regulardictionaries` with a JSON file in the body as follows. <--EM: Is this what is described below? And "in the body" of what?->
```
Comment (RahulReddyRavipally) [inline README.md]: **Is this what is described below? And "in the body" of what?**

Body of the POST request.
Any request comprises of two basic things HEADER and BODY.

File: README.md [hunk 4]
```diff
@@ -90,457 +90,467 @@ type: tutorial
...
 
->NOTE: You will have the `<eddi-url>` generated in [step 1](#step-1-install-the-eddi-operator-from-red-hat-marketplace-on-openshift-cluster).
+>Note: You will use the `<eddi-url>` generated in [step 1](#-step-1-configure-a-red-hat-openshift-cluster-with-red-hat-marketplace).
 
-- In Postman, enter the `<eddi-url>` followed by `/regulardictionarystore/regulardictionaries`, make sure you select the following parameters:
+In Postman, enter the `<eddi-url>` followed by `/regulardictionarystore/regulardictionaries`. Make sure you select the following parameters: <!--EM: What is Postman? How do they get it? Should that in the prereq?-->
```
Comment (RahulReddyRavipally) [inline README.md]: **What is Postman? How do they get it? Should that in the prereq?**

Postman is a collaboration platform for API development. Its an Open Source tool. Yes, this is a prereq for this tutorial.

File: README.md [hunk 5]
```diff
@@ -90,457 +90,467 @@ type: tutorial
...
+    ![](doc/source/images/regulardictionary.png)
 
->Note: The `<UNIQUE_DICTIONARY_ID>` will be used in Packaging, please make a note of it.
+>Note: The `<UNIQUE_DICTIONARY_ID>` will be used to package the elements of the chatbot, so please make a note of it.
 
-#### Step 4.2: Create Behavior Rules
+#### Step 4.2: Create behavior rules
 
-- Behavior Rules are Actions based on decision making with predefined as well as custom conditions.
+*Behavior rules* are actions that are based on decision-making with predefined as well as custom conditions. <!--EM: So, basically, behavior rules are rules that how the chatbot will respond to answers? or where they will look for answers or all of the above? When I read the current definition, I think: Whose decision-making?? the machines?-->
```
Comment (RahulReddyRavipally) [inline README.md]: **So, basically, behavior rules are rules that how the chatbot will respond to answers? or where they will look for answers or all of the above? When I read the current definition, I think: Whose decision-making?? the machines?**

Behavior rules are a series of defined rules by the developer of the bot. These rules are used to answer to the queries asked by the user.

File: README.md [hunk 6]
```diff
@@ -964,87 +976,90 @@ spec:
...
+1. To see the results of the commands you ran in the earlier steps, access the E.D.D.I dashboard using the URL you obtained in [step 1](#step-1-install-the-e.d.d.i-operator-from-red-hat-marketplace-on-openshift-cluster).
 
-- In the dashboard click on `Go to Bot Manager` as shown.
+1. In the dashboard, click on **Go to Bot Manager** as shown.
 
-![](doc/source/images/eddidashboard.png)
+    ![](images/eddidashboard.png)
 
-- In the Bot Manager you can see the Conversation ID of the chatbot that you deployed through API calls.
+1. In the Bot Manager,<!-EM: Is this in E.D.D.I dashboard?--> you can see the Conversation ID of the chatbot that you deployed through API calls.
```
Comment (RahulReddyRavipally) [inline README.md]: **Is this in E.D.D.I dashboard?**
Yes its the EDDI dashboard.

**Comment by RahulReddyRavipally** [comment]
**My one big problem is that I don't see where the reader is accessing COVID-19 information. Covid-19 isn't even mentioned until the second-to-last step where the reader names their chatbot. Is that part of the JSON file? We need to be mentioning that more if that's the heading in the tutorial. It's not clear to me reading these steps what this has to do with COVID-19.**

The rules that are set up is Covid-19 related. This is a covid assessment tool, it access you on various parameters through an interactive way using chatbot. Hence you don't find the work "covid" as often.

**Also, note my question about Postman. That's just sort of randomly introduced. Should it be in the prereqs?**

Yes postman is a prereq.

**Review by emilyannmitchell — COMMENTED** [review]
I made some changes based on your feedback. Please reivew/approve/merge and then I will take that clean file and move it to staging.

**Pull request merged**